GitUI/UserControls/RevisionGrid.cs

```diff
@@ -427,6 +428,78 @@ private void RevisionsKeyDown(object sender, KeyEventArgs e)
             }
         }
 
+        private void RevisionsKeyUp(object sender, KeyEventArgs e)
+        {
+            if (Revisions.RowCount < LastRowIndex || LastRowIndex < 0 || Revisions.RowCount == 0)
```

Comment:
I checked and there are a few instances of this or nearly identical code in use.
I am happy to refactor it as a separate PR but it doesn't feel right to bundle it in here